lib/OpenQA/Scheduler.pm

```diff
@@ -36,6 +36,8 @@ our $RUNNING;
 sub startup {
     my $self = shift;
 
+    OpenQA::Scheduler::Client::mark_current_process_as_scheduler;
+
```

Comment:
Always these additional blank lines …

Reply:
Always these complaints about blank lines ...

Reply:
yes, they won't go away so easily :) So would you mind removing them?

Reply:
The line is pretty special, so the extra blank line makes sense to me. I'd actually rather get rid of the next blank line after this.